### PR TITLE
Add cookie sharing sample projects

### PR DESCRIPTION
**Pull request by guardrex**
@HaoK

RE: Hosting cookie sharing sample projects in the AuthSamples repo. https://github.com/aspnet/Docs/pull/4981

* Ok to group these into one folder given they go together?
* Does the netfx project require any unmagical strings? (i.e., I set the `Version` in the package references of the Core projects, but idk what to do with the netfx project.)
* Is my project naming ok? I kind'a like my existing names. The main folder perhaps could be `Cookies.Interop`, but "Interop" makes it less grokable:tm: than just "CookieSharing."

### CODE REVIEW

**Comment by HaoK** [comment]
I think keeping them together in a subfolder is fine, don't worry too much about making the projects build against the dev branches, you can just add the projects in a state where they build/run for you in VS, and I'll take care of converting them to building against the right dev feeds.

**Comment by guardrex** [comment]
> you can just add the projects in a state where they build/run for you in VS, and I'll take care of converting them to building against the right dev feeds.

so ... revert those `Version` strings back to `2.0.0`?

Also, what's the CI build failure here?

**Comment by HaoK** [comment]
For naming:

CookieSharing is fine, but I'd use that throughout, so something like this as the namespaces/projects

CookieSharing.Core
CookieSharing.WithIdentity.Core
CookieSharing.WithIdentity.NetFramework

**Comment by guardrex** [comment]
Ok ... I'll do that now. I also just realized that I didn't branch properly. Trying to do like three things at once here **did not pay off this afternoon!** lol

File: samples/CookieSharing/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# Cookie Sharing Sample App
+
+The sample illustrates cookie sharing across three apps that use cookie authentication:
+
+* ASP.NET Core 2.0 Razor Pages app without using ASP.NET Core Identity
```
Comment (HaoK) [inline samples/CookieSharing/README.md]: I'd define the exact final project names here and include these long form description of what they are

**Comment by guardrex** [comment]
@HaoK I'm going to resubmit this properly ... right after I close about 20 of these 23 open browser tabs.

> I'd define the exact final project names here and include these long form description of what they are

I'll get that, too.

New PR coming soon!

**Comment by HaoK** [comment]
Don't worry about the CI, as long as the projects work locally for you and run its fine, I'll take care of making sure everything is converted to the CI project system before merging

**Pull request closed without merging**